docs/_templates/indexcontent.html

```diff
@@ -0,0 +1,47 @@
+{% extends "defindex.html" %}
+
+{% block body %}
+
+<div class="figure align-center" id="id1">
+    <img src="_images/logo.svg">
+    <h1>Kinto — Store, Sync, Share and Self-Host.</h1>
+    <p>Kinto is a lightweight JSON storage service, with synchronisation and sharing abilities</p>
+</div>
+
+<div id="toc">
+    <div class="item">
+          <h3><a href="{{ pathto("overview") }}">Overview</a></h3>
+          <p class="description">Get a high level overview of what Kinto is.</p>
+          <p class="links"><a href="{{ pathto("rationale") }}#use-cases">Use cases</a> | <a href="{{ pathto("rationale") }}#comparison">Comparison table</a> | <a href="{{ pathto("overview") }}#faq">FAQ</a></p>
+    </div>
+    <div class="item">
+        <h3><a href="{{ pathto("tutorials/index") }}">Tutorials</a></h3>
+        <p class="description">The easiest way to get started.</p>
+        <p class="links"><a href="{{ pathto("tutorials/first-steps") }}">Store your data</a> | <a href="{{ pathto("tutorials/permissions") }}">Share between users</a> |  <a href="{{ pathto("tutorials/app-examples") }}">Application examples</a></p>
+    </div>
+    <div class="item">
+        <h3><a href="{{ pathto("concepts") }}">Concepts</a></h3>
+        <p class="description">Learn the concepts.</p>
+        <p class="links"><a href="{{ pathto("concepts") }}#buckets-collections-and-records">Buckets, Collections and Records</a> | <a href="{{ pathto("concepts") }}#inherited">Permissions inheritance</a> | <a href="{{ pathto("concepts") }}#groups">Groups</a></p>
+    </div>
+    <div class="item">
+        <h3><a href="{{ pathto("api/index") }}">APIs</a></h3>
+        <p class="description">Extensive HTTP endpoints documentation.</p>
```

Comment:
We can assume that it's documentation. :smile: Consider something like:
> An exhaustive look at our HTTP endpoints.
A deep dive into the HTTP endpoints.
The gritty details of the HTTP endpoints.